packages/trader/src/Modules/Trading/Containers/trade.tsx

```diff
@@ -0,0 +1,235 @@
+            })}
+            id='trade_container'
+        >
+            <DesktopWrapper>
+                <PositionsDrawer />
+            </DesktopWrapper>
+            {/* Div100vhContainer is workaround for browsers on devices
+                    with toolbars covering screen height,
+                    using css vh is not returning correct screen height */}
+            <Div100vhContainer
+                className='chart-container'
+                height_offset={chart_height_offset}
+                id='chart_container'
+                is_disabled={isDesktop()}
+            >
+                <NotificationMessages />
+                <React.Suspense
+                    fallback={<ChartLoader is_dark={is_dark_theme} is_visible={!symbol || !!is_chart_loading} />}
+                >
+                    <DesktopWrapper>
+                        <div className={classNames('chart-container__wrapper', { 'vanilla-trade-chart': is_vanilla })}>
+                            <ChartLoader is_visible={is_chart_loading || should_show_active_symbols_loading} />
+                            <TradeChart topWidgets={topWidgets} is_accumulator={is_accumulator} />
+                        </div>
+                    </DesktopWrapper>
+                    <MobileWrapper>
+                        <ChartLoader is_visible={is_chart_loading || should_show_active_symbols_loading} />
+                        <SwipeableWrapper
+                            className={classNames({ 'vanilla-trade-chart': is_vanilla })}
+                            is_disabled={
+                                !show_digits_stats ||
+                                !is_trade_enabled ||
+                                form_components.length === 0 ||
+                                is_chart_loading ||
+                                should_show_active_symbols_loading
+                            }
+                            is_swipe_disabled={swipe_index === 1}
+                            onChange={onChangeSwipeableIndex}
+                        >
+                            {show_digits_stats && <DigitsWidget digits={digits} tick={tick} />}
+                            <TradeChart
+                                is_accumulator={is_accumulator}
+                                bottomWidgets={show_digits_stats ? bottomWidgets : undefined}
+                                topWidgets={topWidgets}
+                            />
+                        </SwipeableWrapper>
+                    </MobileWrapper>
+                </React.Suspense>
+            </Div100vhContainer>
+            <div className={form_wrapper_class}>
+                {is_market_closed && !is_market_unavailable_visible && (
+                    <MarketIsClosedOverlay
+                        is_eu={is_eu}
+                        is_synthetics_trading_market_available={is_synthetics_trading_market_available}
+                        onClick={onTryOtherMarkets}
```

Comment:
I removed `{...(is_eu && category)}` line from here since it has no effect. It remained after [this change](https://github.com/binary-com/deriv-app/pull/6910).